### ISSUE TITLE
Identify the cause for not working "send" button for prescription exchange from doctor to patient.

### ISSUE THREAD

**Issue by PratikMane0112** [PratikMane0112/TelMedSphere#4]
### Description

When the appointment & video consultation meet is finished. Then doctors has ability to write prescription for patients. They can download it & can send it to patients. It is working fine with downloading but not working while sending to patients. As doctor click on send button it not received at patient side.

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/PratikMane0112/TelMedSphere/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project

### Code of Conduct

- [X] I agree to follow this project "[Code of Conduct](https://github.com/PratikMane0112/TelMedSphere/blob/main/.github/CODE_OF_CONDUCT.md)"

**Comment by PratikMane0112** [comment]
Welcome to 💖TelMedSphere !!!!!!! Thank you for your interest, please go ahead.

**Comment by abhirajadhikary06** [comment]
Thanks for the clear explanation. I will try to work on this but if I am unable to I will inform you early wont delay for sure.

**Comment by abhirajadhikary06** [comment]
Currently I am unable to solve this and need more time. My exams are round the corner too. So I am unassigning from this issue, please assign someone. As my exam ends I will be working on if this is remains unsolved.
Thank you!

**Comment by PratikMane0112** [comment]
Hey @abhirajadhikary06, its okay, really this issue is hard to solve but ya its fine. Be focused on your exams that is your first priority now. Also you can work on other newbie friendly issues which are thier in project or I will tell you. Also I am going to register this project for SWOC and DWOC which are taking part in jan & feb. You can contribute their also. Just let me now when your exam gets over. All the best for exams 👍

**Comment by abhirajadhikary06** [comment]
Yeah sure will be contributing to take forward this project. See you in
SWOC then.
Thanks for understanding

**Comment by Rucha-Ambaliya** [comment]
I’d like to take this up and start working on it. If I’m unable to detect it, I’ll let you know within a few days. I’ll also make sure to keep you updated on the progress regularly.

**Comment by PratikMane0112** [comment]
Hey 

> @PratikMane0112 I’d like to take this up and start working on it. If I’m unable to detect it, I’ll let you know within a few days. I’ll also make sure to keep you updated on the progress regularly.

Thanks, but I would suggest to look over issue of tailwind css first #35 . It has more priority than this. After completing that i will assign this to you.
